### ISSUE TITLE
Menu - drilldown breadcrumb example has unnecessary LI element

### ISSUE THREAD

**Issue by kmcfaul** [patternfly/patternfly-react#7756]
When going between the breadcrumb and menu sections while a menu is drilled in, multiple keypresses of Shift+Tab are required to navigate back to the breadcrumbs. What I think might be happening is that focus is shifting back through the parent menus in the DOM which exist but are not visible before hitting the breadcrumb section again. We probably need to manually move focus for this use case instead of relying on default browser behavior.

EDIT from @thatblindgeye : updating this issue to reflect that the breadcrumb drilldown example has a divider wrapped in an `li` element, without an `ul` or `ol` element containing it.

**Comment by nicolethoen** [comment]
There's a divider between the breadcrumb and the menu that shouldn't be wrapped in an `<li>`
